x/concentrated-liquidity/export_test.go

```diff
@@ -1,6 +1,6 @@
 package concentrated_liquidity
 
 // SetInitialPoolDenoms sets the pool denoms of a cl pool
-func (p *Pool) SetInitialPoolDenoms(poolDenoms []string) error {
-	return p.setInitialPoolDenoms(poolDenoms)
+func (p *Pool) SetInitialPoolDenoms(denom0, denom1 string) error {
```

Comment:
Both line 3 and 4 should be called `OrderInitialPoolDenoms`

Reply:
changed as well